rockcraft/extensions/__init__.py

```diff
@@ -26,3 +26,7 @@
     "register",
     "unregister",
 ]
+
+from .paas_flask import PaasFlask as _PaasFlask
+
+register("paas-flask", _PaasFlask)
```

Comment:
this should be called flask, which is what we agreed to (no paas in the name)

Reply:
Updated, thanks!